src/moveTo.js

```diff
@@ -8,7 +8,7 @@ const MoveTo = (() => {
     tolerance: 0,
     duration: 800,
     easing: 'easeOutQuart',
-    callback: function() {},
```

Comment:
This is an eslint rule called the [comma-dangle](http://eslint.org/docs/rules/comma-dangle). This should not be erased.